assets/queries/common/passwords_and_secrets_in_infrastructure_code/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "f996f3cb-00fc-480c-8973-8ab04d44a8cc",
+  "queryName": "Passwords And Secrets In Infrastructure Code",
+  "severity": "HIGH",
+  "category": "Network Security",
+  "descriptionText": "Query to find passwords and secrets in infrastructure code.",
+  "descriptionUrl": "#"
```

Comment:
This will fail in unit tests after this PR https://github.com/Checkmarx/kics/pull/1841 gets merged. We should provide a valid URL.